### DISCUSSION TITLE
Issue with resize, pixelated rows

### DISCUSSION THREAD

**Discussion by Lightsparker** [ImageMagick/ImageMagick#2238] [Help] [answered]
Im using this code in windows to resize images in a directory to several sizes. Original images aspect ratio are 1:1.41, dimension are 5120x7240
`convert *.jpg ( +clone -set filename:original %t -resize 2560 -quality 82 -write %[filename:original]-2560.jpg ) ( +clone -set filename:original %t -resize 1920 -quality 82 -write %[filename:original]-1920.jpg ) ( +clone -set filename:original %t -resize 1366 -quality 82 -write %[filename:original]-1366.jpg ) ( +clone -set filename:original %t -resize 1080 -quality 82 -write %[filename:original]-1080.jpg ) ( +clone -set filename:original %t -resize 720 -quality 82 -write %[filename:original]-720.jpg ) null:`
So when Im resizing to widths 2560, 1920, 1366, the result is good and no problem, but in width sizes 1080, 720, pixelated lines are shown.
![pixelatedline](https://user-images.githubusercontent.com/67903796/86587317-a5b8d500-bf89-11ea-8e3c-bdf69bd602a9.jpg)
I think the problem is related to aspect ratio but I don't know how to solve it

**Comment by snibgo** [comment, marked as answer]
That's a very strange command. You read all your jpg files, but only process the last one.

As you write to the same directory you read from, running the command twice will process the results of the previous run. I don't suppose you want that.

You make an image, resized to 2560. Then you resize the result to 1920. Then you resize that result to 1366, and so on. This is like a photocopy of a photocopy of a photocopy. I suggest instead that you should resize the _original_ to 2560, then resize the _original_ to 1366, and so on.

And when you have written an image, you don't need to keep it in the list, so delete it.

**Reply by Lightsparker** [reply]
lol the truth is that I'm new to this and I still don't know what I'm doing to be honest. I'm still trying to improve the code, but thank you a lot I understand everything a little better
